### ISSUE TITLE
separate-close-popup

### ISSUE THREAD

**Issue by bortomar** [quasarframework/quasar#7028]
QMenu option "separate-close-popup" is not working since version 1.11.1 for nested menus.

```html 
<q-btn label="menu#1">
        <q-menu>
            <q-list>
                <q-item clickable>item#1.1</q-item>
                <q-item>     
                <q-btn label="menu#2">   
                    <q-popup-proxy>
                        <q-list>
                            <q-item clickable v-close-popup="1">item#2.1</q-item>
                        </q-list>
                    </q-popup-proxy>
                </q-btn>
                </q-item>
            </q-list>
        </q-menu>
      </q-btn>
```

The following statement from v-close-popup directive documentation is also no longer true.
> QPopupProxy separates chained menus

>

**Comment by pdanpdan** [comment]
Can you please make a codepen?

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.11.2.

**Comment by mastamitche** [comment]
Can we reopen this?

It's happening again.

https://codepen.io/bortomar/pen/wvKxjxN?editors=1010
